src/backend/TrafficCourts/Messaging/MessageContracts/DisputeApproved.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TrafficCourts.Messaging.MessageContracts
+{
+    public interface DisputeApproved: Message
```

Comment:
I think interfaces are supposed to start with an 'I' in C#.
```suggestion
    public interface IDisputeApproved: Message
```

Reply:
Ah, okay, very good.